.gitignore

```diff
@@ -1,2 +1,3 @@
 __pycache__
 /sandbox
+.idea
```

Comment:
Oh, I started trying out PyCharm after a conversation with John yesterday and this is from that.  If there's no objection, I'm inclined to leave it in.

Reply:
I have no complaints about having it